data/remnant/remnant events.txt

```diff
@@ -1182,6 +1182,9 @@ event "remnant: teciimach deployment"
 		add "Teciimach Pod"
 		add "Teciimach Canister"
 		add "Teciimach Canister Rack"
+	outfitter "Remnant Core"
+		remove "EMP Torpedo"
```

Comment:
```suggestion
```

Leaving it here is acceptable with the lore, if people want to keep access.

Reply:
Rephrasing my previous comment: At least a few seem to want the EMP to remain accessible somewhere, so I would recommend committing that suggestion to remove the `remove "EMP Torpedo"`.